packages/connection-model/lib/model.js

```diff
@@ -1046,14 +1048,23 @@ Connection = AmpersandModel.extend({
 const parseConnectionStringAsPromise = promisify(parseConnectionString);
 
 async function createConnectionFromUrl(url) {
+  // We use resolveMongodbSrv because it understands the load balancer
+  // option, whereas parseConnectionString from the 3.6 driver does not.
+  // This could potentially go away once we're using the 3.7 driver,
```

Comment:
// This could potentially go away once we're using the 3.7 driver,

At this point we won't use 3.7, is there a chance this is going to go away with some 4.x?

Reply:
There is no `parseOptions` like the one we use here in the 4.x driver, otherwise we would use it :)